catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/ResourceOperations.java

```diff
@@ -817,4 +780,44 @@ protected void validateGetResourceRequest(ResourceRequest getResourceRequest)
           "Cannot perform getResource with null attribute value or null attributeName, either passed in from endpoint, or as output from PreResourcePlugin");
     }
   }
+
+  // Ignoring cognitive complexity to maintain error catching
```

Comment:
✏️ I think you can reduce the cognitive complexity while still catching errors.  Here are some articles that I think might help.

https://testing.googleblog.com/2017/06/code-health-reduce-nesting-reduce.html
https://chrismm.com/blog/writing-good-code-reduce-the-cognitive-load/